creator-node/src/snapbackSM/syncHistoryAggregator.js

```diff
@@ -151,8 +154,8 @@ class SyncHistoryAggregator {
   static async getLatestSyncData (logContext = {}) {
     const logger = genericLogger.child(logContext)
     let currentLatestSyncData = {
-      success: null,
-      fail: null
+      success: 0,
+      fail: 0
```

Comment:
oh i didn't realize you'd replicated this behavior, didn't even realize this was null. fair to change, hopefully doesn't affect its consumption though

Reply:
it doesnt. checked on stage again 😄

Reply:
legend